### PR TITLE
Convert & update the Getting Started Wiki to MD.

### PR DESCRIPTION
**Pull request by mars-lan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,11 +59,11 @@ Want help? Check out the [Gitter chat room][GITTER] and [Google Groups][LIST]
 
 
 [wiki]: https://github.com/LinkedIn/Wherehows/wiki
-[GS]: https://github.com/LinkedIn/Wherehows/wiki/Getting-Started
+[GS]: https://github.com/linkedin/WhereHows/blob/master/wherehows-docs/getting-started.md
 [CON]: https://github.com/LinkedIn/Wherehows/wiki/Contributing
 [VM]: https://github.com/LinkedIn/Wherehows/wiki/Quick-Start-With-VM
```
Comment (alyiwang) [inline README.md]: Do we want to deprecate the VM?

File: wherehows-docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+Frontend is a standard Ember.js web app running on top of a Play server. Please refer to [Ember](https://www.emberjs.com/) & [Play](https://www.playframework.com/documentation) documents for more details on these frameworks.
+
+### Configuration
+All the frontend-specific configurations are stored in [wherehows-frontend/conf/application.conf](https://github.com/linkedin/WhereHows/blob/master/wherehows-frontend/conf/application.conf). You'll need to change it to match your environment.
+
+### Development
+Run the following command to launch an instance of frontend service locally for development.
+
+    cd wherehows-frontend
+    ../gradlew runPlayBinary
```
Comment (alyiwang) [inline wherehows-docs/getting-started.md]: " ../gradlew run " will also do

File: README.md (hunk 1 above)
Comment (mars-lan) [inline README.md]: We don't have a demo docker image yet so can't fully deprecate VM

File: wherehows-docs/getting-started.md (hunk 2 above)
Comment (mars-lan) [inline wherehows-docs/getting-started.md]: Yes, but that relies on gradle task name abbreviation: https://docs.gradle.org/3.3/userguide/tutorial_gradle_command_line.html#sec:task_name_abbreviation, which is less clear

**Pull request merged**